tests/MongoDB.Driver.Core.Tests/Specifications/read-write-concern/ConnectionStringTestRunner.cs

```diff
@@ -76,8 +76,8 @@ private void AssertValid(ConnectionString connectionString, BsonDocument definit
             BsonValue writeConcernValue;
             if (definition.TryGetValue("writeConcern", out writeConcernValue))
             {
-                var writeConcern = WriteConcern.FromBsonDocument(MassageWriteConcernDocument((BsonDocument)writeConcernValue));
-
+                var writeConcern =
+                        WriteConcern.FromBsonDocument(MassageWriteConcernDocument((BsonDocument) writeConcernValue));
```

Comment:
Minor comment on whitespace: I think we tend to prefer a four space indent for multiline statements :)